met-web/src/components/engagement/view/EngagementView.tsx

```diff
@@ -59,10 +60,12 @@ export const EngagementView = () => {
                     direction="row"
                     justifyContent={'flex-start'}
                     alignItems="flex-start"
-                    sx={{ margin: '1em 2em 1em 3em' }}
-                    m={{ lg: '0 8em 1em 3em', md: '2em', xs: '1em' }}
+                    m={{ md: '2em', xs: '1em' }}
                     spacing={2}
                 >
+                    <Grid item xs={12}>
+                        <PhasesWidget />
```

Comment:
i am not sure if we should change , but i have a very slight inclination towards calling it as eaophasewidget since its just eao specific